packages/search/src/style.scss

```diff
@@ -7,7 +7,7 @@
 @import "@wordpress/base-styles/mixins";
 @import "@wordpress/base-styles/breakpoints";
 
-$input-z-index: 20;
+$input-z-index: 0;
```

Comment:
@tyxla @hambai this change looked fine in my testing but I'd like you to test the places where this is used if possible.

The site selector and site picker include it. You can search for `'@automattic/search'` in the codebase to look for other usages.